core/src/main/java/kafka/server/SharePartition.java

```diff
@@ -955,7 +955,7 @@ private NavigableMap<Long, InFlightBatch> fetchSubMapForAcknowledgementBatch(
             if (floorOffset == null && hasStartOffsetMoved) {
                 // If the start offset is between the first and last offset of the acknowledgment batch, then
                 // we need to get the floor offset using the last offset of the acknowledgment batch.
```

Comment:
I think perhaps the comment is now misleading.

Reply:
yeah, sorry, forgot to update it, doing it now